.circleci/config.yml

```diff
@@ -108,6 +106,51 @@ jobs:
           paths:
             - release
 
+  windows:
+    executor:
+      name: win/default
+    working_directory: C:\Users\circleci\simplenote-electron
+    environment:
+      CSC_FOR_PULL_REQUEST: true
+    steps:
+      - checkout
+      - attach_workspace:
+          at: C:\Users\circleci\simplenote-electron
+      - run:
+          name: Install make
+          command: cinst make
+      - run:
+          name: Install node version
+          command: |
+            $NODE_VERSION = Get-Content .\calypso\.nvmrc
+            nvm install $NODE_VERSION
+            nvm use $NODE_VERSION
+      - run:
+          name: Npm install
+          command: |
+            # The release of nodejs currently being used by Calypso includes a buggy version 
```

Comment:
Can probably update these comments so they don't refer to Calypso. 

It might also be worth verifying whether or not you need this fix at all (i.e. if the version of Node you're building with doesn't need this patch, then you might as well leave it out).

Reply:
https://github.com/Automattic/simplenote-electron/pull/1771/commits/f2055d78a0cd4796947cb0d113176be749467e02